src/constants/apiPaths.ts

```diff
@@ -2,7 +2,7 @@ const API_PATHS = {
   product:
     "https://zlszw61kh8.execute-api.eu-west-1.amazonaws.com/dev/products",
   order: "https://.execute-api.eu-west-1.amazonaws.com/dev",
-  import: "https://.execute-api.eu-west-1.amazonaws.com/dev",
+  import: "https://bv408uyq3d.execute-api.eu-west-1.amazonaws.com/dev",
```

Comment:
Hi @nastarash Anastasiya Rashchynskaya
Please remember to update the new import link as you wrote in the description, this link below is valid for now :)

https://7jki5muxi1.execute-api.eu-west-1.amazonaws.com/dev/import

Reply:
Fair point, thx for mentioned that
Links was updated and FE app was redeployed